x/feemarket/keeper/params.go

```diff
@@ -30,8 +30,12 @@ func (k Keeper) GetParams(ctx sdk.Context) types.Params {
 	objStore := ctx.ObjectStore(k.objectKey)
 	v := objStore.Get(types.KeyPrefixObjectParams)
 	if v == nil {
+		store := k.storeService.OpenKVStore(ctx)
```

Comment:
What's the difference here, using store service vs store key?

Reply:
return is different, and we need panic if err right?

Reply:
the behavior identical, the old store also panic on invalid input.